library/common/config_template.cc

```diff
@@ -17,13 +48,13 @@ const char* config_template = R"(
                 address:
                   socket_address: {address: {{ domain }}, port_value: 443}
     tls_context: &base_tls_context
-      common_tls_context:
+      common_tls_context: &base_common_tls_context
         validation_context:
           trusted_ca:
             inline_string: |
 )"
 #include "certificates.inc"
-R"(
+                              R"(
```

Comment:
whitespace?

Reply:
my local formatter changes this. But it is a noop.